canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -1011,6 +1040,23 @@ export default class PropertiesController {
 			this.propertiesStore.updatePropertyValue(propertyId, initialValue);
 			return;
 		}
+		// evaluate conditional defaults upon each control input change
+		if (this.form) {
+			let propertyValues = {};
```

Comment:
Why do we need this along with the 161-182 above?  They seem to be doing the same thing.

Reply:
161-182 - Called initially when properties are loaded. This is called only once. 

1043-1059 - Called after properties value is changed, check if condition is still true. This can be called multiple times.
For example - condition checks if `mode = Include`. If I update the value of `mode`, condition should return false.